deployment/scripts/install.sh

```diff
@@ -277,11 +277,11 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete serviceaccount,service,route,oauthclient --selector logging-infra=support
```

Comment:
The oc delete will handle it. We could do it by specific name, but I prefer to leave specifics in templates as much as possible.